Hydrogent/src/HnRenderDelegate.cpp

```diff
@@ -263,7 +263,7 @@ HnRenderDelegate::HnRenderDelegate(const CreateInfo& CI) :
     m_PrimitiveAttribsCB{CreatePrimitiveAttribsCB(CI.pDevice)},
     m_MaterialSRBCache{HnMaterial::CreateSRBCache()},
     m_USDRenderer{CreateUSDRenderer(CI, m_PrimitiveAttribsCB, m_MaterialSRBCache)},
-    m_TextureRegistry{CI.pDevice, CI.TextureAtlasDim != 0 ? m_ResourceMgr : nullptr},
+    m_TextureRegistry{CI.pDevice, CI.TextureAtlasDim != 0 ? m_ResourceMgr : RefCntAutoPtr<Diligent::GLTF::ResourceManager>(nullptr)},
```

Comment:
`Diligent` is not needed here as the code is already in the namespace.
`RefCntAutoPtr` has a default ctor:
`RefCntAutoPtr<GLTF::ResourceManager>{}`

Reply:
Done